### DISCUSSION TITLE
Basic Example

### DISCUSSION THREAD

**Discussion by PiusHohenberg** [ansys/pydpf-core#809] [Q&A] [answered]
Hi,
just installed pyAnsys using: pip install pyansys on a local system (Windows 10, Ansys 2021 R2).
The basic example is:

from ansys.dpf import core as dpf
from ansys.dpf.core import examples
model = dpf.Model(examples.find_simple_bar())
print(model)

This gives an error message because find_simple_bar() can not be found. Is this intentionally so? Am I doing something wrong?
Thanks & Greetings

**Comment by PProfizi** [comment]
Hi @PiusHohenberg, 
A first answer I can give you is that AFAIK the pyansys package started tracking compatible pyansys packages for Ansys 2023R1, so what you installed might not be exactly right for use with Ansys 2021 R2 (for example, for PyDPF-Core, the compatibility table is given [here](https://dpf.docs.pyansys.com/getting_started/compatibility.html)).
I can however guide you to the right PyDPF package versions but I would need for you to share the list of installed packages in your environment.

**Reply by PProfizi** [reply]
@PiusHohenberg I checked and `examples.find_simple_bar()` is available starting with ansys-dpf-core 0.6.2 so this might explain why your example does not work. The latest pyansys package (v2023.1.3) does indeed install ansys-dpf-core 0.6.0.

**Comment by PiusHohenberg** [comment, marked as answer]
Thank you for your support. While looking into some scirpt files I found
the older way to call the basic example. I can now read result files
created on my local machine (Ansys 2021R2). However, reading files created
on a HPC cluster using Ansys 2022R1 still gives me errors. Is this an
expected behaviour?

Am Do., 23. Feb. 2023 um 17:32 Uhr schrieb PProfizi <
***@***.***>:
